modules/plugin/geotiff/src/test/java/org/geotools/gce/geotiff/GeoTiffReaderTest.java

```diff
@@ -1150,4 +1151,34 @@ public void testFileDoesNotExist() throws Throwable {
             file.setReadable(true);
         }
     }
+
+    /**
+     * The GeoTiffReader should be able to read from an InputStream
+     */
+    @Test
+    public void testCanReadInputStream() throws IOException {
+        File rasterfile = TestData.file(GeoTiffReaderTest.class, "geo.tiff");
+        FileInputStream is = new FileInputStream(rasterfile);
+
+        // Read coverage
+        GeoTiffReader reader = new GeoTiffReader(is);
```

Comment:
Always remember to dispose of the reader like in the other tests, there might be build issues in Windows otherwise (due to file locking)